app/views/bookmarks/show.html.erb

```diff
@@ -1,11 +1,22 @@
 <div class="row">
   <div class="col-md-9">
-    <h4><a href= <%= @url %> > <%= @url %> </a> </h4>
+    <h4><a target="_blank" href= <%= @url %> > <%= @url %> </a><h4>
```

Comment:
Fix href attribute so it is wrapped with quotes when you right-click "View source" the page in chrome.